### PR TITLE
Menu bar

### PR DESCRIPTION
**Pull request by DanBotero**
Clean the menu bar and the buttons for the home button.

### CODE REVIEW

**Comment by kirjs** [comment]
Screenshot?

**Comment by DanBotero** [comment]
Sure...
screen shot 2017-05-04 at 8 52 18 pm
and the code was changed too. Super small things. Also, there are new comments from my class.

File: src/app/presentation/menu-shortcut/menu-shortcut.component.html [hunk 1]
```diff
@@ -5,12 +5,13 @@
         <p><a href="/">Index</a></p>
       </li>
       <li class="menu-item" *ngFor="let route of routes">
-        <p><a [href]="route.path + '/0'">{{route.name}}</a></p>
+        <p><a [href]="route.path">{{route.name}}</a></p>
```
Comment (kirjs) [inline src/app/presentation/menu-shortcut/menu-shortcut.component.html]: can you test this? Open the thing, and check that once you go to the next slide URL is udpated. 

Also replace href with routerLink for #86

**Comment by prestonvanloon** [comment]
Can this be merged @kirjs ? It has approval

**Pull request merged**